src/vue/backend/src/serviceContainer.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Interface to adhere to.
+ */
+const RepositoryInterface = {
+    get () {},
+    post () {}
+}
+
+/**
+ * Dynamically load a repository file and bind it to the Repository interface.
+ * Note: The file must contain .class - ex: myfile.class.js
+ *
+ * @param {*} fileName The name of the repository file.
+ */
+const loadRepo = (fileName) => {
+    /* Using template literals breaks the dynamic import. /shrug. */
+    return bind(() => import('./repositories/' + fileName + '.class'), RepositoryInterface)
+}
+
+const bind = (repositoryFactory, Interface) => {
+    return {
+        ...Object.keys(Interface).reduce((prev, method) => {
+            const resolveableMethod = async (...args) => {
```

Comment:
`args` is destructured here then spread in the return without being used otherwise, can just pass `args` without destructing it instead.